### ISSUE TITLE
Mark Hazelcast dependency as optional

### ISSUE THREAD

**Issue by keesvandieren** [stevensouza/jamonapi#8]
We are using Jamonapi, but not Hazelcast.

Jamon has a dependency on hazelcast, but still works well when excluding it. (at least in the way we use it).

Can it be marked optional in https://github.com/stevensouza/jamonapi/blob/master/jamon/pom.xml

E.g. this way:

```
  <dependency>
      <groupId>com.hazelcast</groupId>
      <artifactId>hazelcast-all</artifactId>
      <version>${hazelcast.version}</version>
      <optional>true</optional>
  </dependency>
```

Thanks in advance.

**Comment by stevensouza** [comment]
You are right it is optional.  I will look into making that more explicit.

Thanks for your help.

Steve

**Comment by kk-in-bg** [comment]
Hello Steve,
How can I get the version of hazelcast-all used in jamonapi? And from where this variable <version>${hazelcast.version}</version> gets the version details?
thank you.
